### DISCUSSION TITLE
部署 linux

### DISCUSSION THREAD

**Discussion by pure-love-soldier** [simple-robot/Discussions#95] [Q&A] [answered]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

大佬们，请问下simbot项目部署到服务器怎么解决滑动验证码的问题呢

**Comment by ForteScarlet** [comment, marked as answer]
在linux上跟在电脑上都一样，不过最好是电脑上操作完了放在服务器上的时候把 device.json 、cache 等内容复制上去。
懒得复制的话，linux上又不是没有验证码链接

**Reply by pure-love-soldier** [reply]
好的，谢谢
